### PR TITLE
(maint) Release prep 0.4.1

### PR DESCRIPTION
**Pull request by rodjek**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+## 0.4.1
+
+- Only include puppet\_litmus on Ruby 2.5 and Ruby 2.6
```
Comment (glennsarti) [inline CHANGELOG.md]: Is that `\_` a typo?

Comment (glennsarti) [reply CHANGELOG.md]: Nope nevermind.  It's escaping the underscore.  Ignore.

**Comment by glennsarti** [comment]
Confirmed the diff.  That's the only changes.

File: CHANGELOG.md (hunk 1 above)
Comment (rodjek) [reply CHANGELOG.md]: Nope, it's a markdown file, `_` should be escaped.

**Pull request merged**